sass_processor/management/commands/compilescss.py

```diff
@@ -84,7 +84,10 @@ def find_templates(self):
             try:
                 module = import_module(loader.__module__)
                 get_template_sources = getattr(module, 'get_template_sources', loader.get_template_sources)
-                paths.update(list(get_template_sources('')))
+                template_sources = list(get_template_sources(''))
+                if template_sources and type(template_sources[0]) is Origin:
```

Comment:
@jrief: No, I didn't even think about it. This should probably cover subclasses, though, so I'll change it to `isinstance`.